cookiecutter/hooks.py

```diff
@@ -50,7 +52,7 @@ def find_hooks():
     return r
 
 
-def run_script(script_path, cwd='.'):
+def run_script(script_path, cwd='.', context={}):
```

Comment:
Since this makes `run_script()` accept a context dict, you might as well combine `run_script()` and `run_script_with_context()` into one function. Then if `run_script_with_context()` is too long, feel free to refactor it somehow if you think it will make it more readable